app/src/test/kotlin/co/netguru/baby/monitor/client/feature/babycrynotification/NotifyBabyCryingUseCaseTest.kt

```diff
@@ -0,0 +1,36 @@
+package co.netguru.baby.monitor.client.feature.babycrynotification
+
+import co.netguru.baby.monitor.RxSchedulersOverrideRule
+import co.netguru.baby.monitor.client.feature.firebasenotification.FirebaseNotificationSender
+import co.netguru.baby.monitor.client.feature.firebasenotification.NotificationType
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.verify
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+class NotifyBabyCryingUseCaseTest {
+
+    @Rule
+    @JvmField
+    val schedulersRule = RxSchedulersOverrideRule()
+
+    private val notificationSender: FirebaseNotificationSender = mock()
+    private lateinit var notifyBabyCryingUseCase: NotifyBabyCryingUseCase
```

Comment:
JUnit classes are always recreated before each test.
https://www.logicbig.com/tutorials/unit-testing/junit/lifecycle.html
Can be replaced just with:
```
private val notifyBabyCryingUseCase = NotifyBabyCryingUseCase(notificationSender)
```

Reply:
Bad habit